### PR TITLE
LRQA-62024-2 | master

### PR DESCRIPTION
**Pull request by PablitoBonito**
https://issues.liferay.com/browse/LRQA-62024

### CODE REVIEW

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by PablitoBonito** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/macros/FrontendSPAInfrastructure.macro [hunk 1]
```diff
@@ -0,0 +1,11 @@
+definition {
+
+	macro Save {
+		Click(locator1 = "Frontend SPA Infrastructure#Save");
+	}
+
+	macro UncheckDisableSPA {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/FrontendSPAInfrastructure.macro:7]: needs to be lowercase uncheckDisableSPA

File: portal-web/test/functional/com/liferay/portalweb/macros/FrontendSPAInfrastructure.macro [hunk 2]
```diff
@@ -0,0 +1,11 @@
+definition {
+
+	macro Save {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/FrontendSPAInfrastructure.macro:3]: can use PortletEntry.save

File: portal-web/test/functional/com/liferay/portalweb/paths/FrontendSPAInfrastructure.path [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">Pawel</td></tr>
+</thead>
+<tbody>
+<tr>
+	<td>Disable_SPA</td>
+	<td>//label[(.='Disable SPA in Internet Explorer')]/input</td>
+	<td></td>
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/paths/FrontendSPAInfrastructure.path:14]: can move to Checkbox.function, rename to ANY_CHECKBOX_SPECIFIC. And make "Disable SPA in Internet Explorer" generic variable for anyone else to use.

File: portal-web/test/functional/com/liferay/portalweb/tests/UncheckDisableSPAInInternetExplorer.testcase [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+			portlet = "System Settings");
+
+		SystemSettings.gotoConfiguration(
+			configurationCategory = "Infrastructure",
+			configurationName = "Frontend SPA Infrastructure",
+			configurationScope = "System Scope");
+
+		FrontendSPAInfrastructure.UncheckDisableSPA();
+
+		FrontendSPAInfrastructure.Save();
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/UncheckDisableSPAInInternetExplorer.testcase:26]: will modify these...use lowercase for macro. ie:   uncheckDisableSPA(); and save();

File: portal-web/test/functional/com/liferay/portalweb/paths/FrontendSPAInfrastructure.path [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+</tr>
+<tr>
+	<td>Save</td>
+	<td>//button[@id='_com_liferay_configuration_admin_web_portlet_SystemSettingsPortlet_save']</td>
+	<td></td>
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/paths/FrontendSPAInfrastructure.path:19]: existing macro we can use. PortletEntry.save();

**Comment by john-co** [comment]
I pushed the review edits to https://github.com/john-co/liferay-portal/pull/353

**Pull request closed without merging**